frontend/src/App.js

```diff
@@ -1,9 +1,23 @@
+import SearchForm from './SearchForm';
+import ResultsList from './ResultsList';
 
 const App = () => (
   <div>
```

Comment:
How about using the short syntax `<> </>` for App instead of div then? I did it like that and then removed it :) That would remove the unnecessary div.
Thanks for the comment!

Reply:
I did not see this comment in time. For the sake of readability, I would recommend you go with the long version of the Fragment component. I requested that change in your PR.